spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/SimpleMessageListenerContainer.java

```diff
@@ -1353,7 +1353,7 @@ protected void invokeListener(Channel channel, Message message) throws Exception
 	 * @throws Exception if the shared connection still can't be established
 	 */
 	protected void handleStartupFailure(BackOffExecution backOffExecution) throws Exception {
-		long recoveryInterval = backOffExecution.nextBackOff();
+		long recoveryInterval = backOffExecution != null ? backOffExecution.nextBackOff() : DEFAULT_RECOVERY_INTERVAL;
```

Comment:
Good; it wasn't obvious to me where the issue was so I just added the protection. 

Pushed